lib/resources/customers.js

```diff
@@ -7,7 +7,7 @@ var customers = module.exports = {
 };
 
 customers.search = requestManager.describe({
-  path: '/v1/customers/search',
+  path: '/v1/customers/search?email=:email',
```

Comment:
Usar la firma generica 
```suggestion
  path: '/v1/customers/search',
```